### PR TITLE
Enhancement: header emphasize actions

### LINKED ISSUE #3017: [Enhancement] Enforce emphasizeActions
- [ ] I have written a [good issue](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Issue) 

## Describe the enhancement
Som jeg forstår det er team kirby snart på vej med en ændring til kirby header (emphasizeActions=true) sådan at man på device kan placere sine knapper/action group under selve header title, men på web bliver de højre centreret ud for title. Vi har dog brug for at de altid er placeret under kirby header title, sådan at vi kan vide hvor stor viewporten bliver til virtual scroll. Vi har derfor brug for at enforce emphasizeActions, sådan at knapper/action group altid er placeret under kirby header title/subtitle underordnet skærmens bredde.

## Describe the solution you'd like
En mulig for at enforce emphasizeActions

<hr />

## Checklist:

The following tasks should be carried out in sequence in order to follow [the process of contributing](https://github.com/kirbydesign/designsystem/blob/main/.github/CONTRIBUTING.md/#the-process-of-contributing) correctly.

### Refinement

- [ ] Request that the issue is [UX refined](https://github.com/kirbydesign/designsystem/blob/main/.github/CONTRIBUTING.md/#ux-refinement); do not proceed until this is done.
- [ ] Request that the issue is [tech refined](https://github.com/kirbydesign/designsystem/blob/main/.github/CONTRIBUTING.md/#tech-refinement); do not proceed until this is done.

### Implementation 
The contributor who wants to implement this issue should: 

- [ ] Make sure you have read: "[Before you get coding](https://github.com/kirbydesign/designsystem/blob/main/.github/CONTRIBUTING.md/#before-you-get-coding)".
- [ ] Signal to others you are working on the issue by assigning yourself.
- [ ] Create a branch from the [develop branch](https://github.com/kirbydesign/designsystem/tree/develop) following our [branch naming convention](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Branch). 
- [ ] Publish a WIP implementation to Github as a draft PR and ask for feedback. 
- [ ] Make sure you have implemented tests following the guidelines in: "[The good: Test](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Test)".
- [ ] Update the [cookbook](https://cookbook.kirby.design) with examples and showcases.

### Review
Once the issue has been implemented and is ready for review:

- [ ] Do a [self-review](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Self-review). 
- [ ] Create a pull-request. If you created a draft PR during implementation you can just mark that as "ready for review".

### LINKED ISSUE #3051: [Enhancement] Header: emphasize actions
- [x] I have written a [good issue](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Issue) 

## Describe the enhancement

As per the [design specs](https://zpl.io/6NOAqeP) it should be possible to configure the header to emphasize the actions.
 
## Describe the solution you'd like

Finish implementation of the `emphasizeActions` property on header and add showcase example and API documentation.

## Have you considered any alternatives?

No.

## Are there any additional context?

No.

<hr />

## Checklist:

The following tasks should be carried out in sequence in order to follow [the process of contributing](https://github.com/kirbydesign/designsystem/blob/main/.github/CONTRIBUTING.md/#the-process-of-contributing) correctly.

### Refinement

- [x] Request that the issue is [UX refined](https://github.com/kirbydesign/designsystem/blob/main/.github/CONTRIBUTING.md/#ux-refinement); do not proceed until this is done.
- [x] Request that the issue is [tech refined](https://github.com/kirbydesign/designsystem/blob/main/.github/CONTRIBUTING.md/#tech-refinement); do not proceed until this is done.

### Implementation 
The contributor who wants to implement this issue should: 

- [x] Make sure you have read: "[Before you get coding](https://github.com/kirbydesign/designsystem/blob/main/.github/CONTRIBUTING.md/#before-you-get-coding)".
- [x] Signal to others you are working on the issue by assigning yourself.
- [x] Create a branch from the [develop branch](https://github.com/kirbydesign/designsystem/tree/develop) following our [branch naming convention](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Branch). 
- [ ] Publish a WIP implementation to Github as a draft PR and ask for feedback. 
- [ ] Make sure you have implemented tests following the guidelines in: "[The good: Test](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Test)".
- [ ] Update the [cookbook](https://cookbook.kirby.design) with examples and showcases.

### Review
Once the issue has been implemented and is ready for review:

- [ ] Do a [self-review](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Self-review). 
- [ ] Create a pull-request. If you created a draft PR during implementation you can just mark that as "ready for review".

### PR DESCRIPTION
**Pull request by jakobe**
## Which issue does this PR close?

This PR closes #3051 and closes #3017

## What is the new behavior?

Adding full support for `emphasizeActions` on Header component.
Also removing internal plumbing for `isResizeable ` as we don't support it for now and need to investigate if we will / it's even needed.

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

## Checklist:

The following tasks should be carried out in sequence in order to follow [the process of contributing](https://github.com/kirbydesign/designsystem/blob/main/.github/CONTRIBUTING.md/#the-process-of-contributing) correctly.

### Reminders
- [ ] Make sure you have implemented tests following the guidelines in: "[The good: Test](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Test)".
- [ ] Make sure you have updated the cookbook with examples and showcases (for bug fixes, enhancements & new components).

### Review  
- [ ] Determine if your changes are a fix, feature or breaking-change, and add the matching label to your PR. If it is tooling, dependency updates or similar, add ignore-for-release.
- [ ] Do a [self-review](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Self-review).
- [ ] Request that the changes are code-reviewed 
- [ ] Request that the changes are [UX reviewed](https://github.com/kirbydesign/designsystem/blob/main/.github/CONTRIBUTING.md/#ux-review) (only necessary if your PR introduces visual changes)

When the pull request has been approved it will be merged to `develop` by Team Kirby.

### CODE REVIEW

File: apps/cookbook/src/app/examples/header-example/examples/emphasize-actions.ts [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  template: `<kirby-page defaultBackHref="/">
+  <kirby-header [emphasizeActions]="true" [title]="'Emphasize Actions'" subtitle1="Subtitle one" subtitle2="Subtitle two">
+    <kirby-action-group *kirbyHeaderActions>
+      <button kirby-button attentionLevel="1" (click)="actionClicked('Action 1')">
+        <kirby-icon name="edit"></kirby-icon>
+        <span class="text">Action 1</span>
+      </button>
+      <button kirby-button attentionLevel="3" (click)="actionClicked('Action 2')">
+        Action 2
+      </button>
```
Comment (RasmusKjeldgaard) [inline apps/cookbook/src/app/examples/header-example/examples/emphasize-actions.ts]: Consider adding an icon to this button or add an additional button so it collapses into a more menu on mobile. 

Support for one button with icon and one without is poor on mobile: 
image

File: libs/designsystem/action-group/src/action-group.component.ts [hunk 2]
```diff
@@ -59,9 +56,6 @@ export class ActionGroupComponent implements AfterContentInit {
   _isCollapsed: boolean;
   _collapsedActions: CollapsedAction[] = [];
 
-  @HostBinding('class.is-resizeable')
-  _isResizeable = false;
```
Comment (RasmusKjeldgaard) [inline libs/designsystem/action-group/src/action-group.component.ts:63]: Consider adding a couple of lines to the PR description about the fact that we are removing the resizable groundwork for now and will re-introduce later.

**Review by RasmusKjeldgaard — CHANGES_REQUESTED** [review]

File: apps/cookbook/src/app/examples/header-example/examples/emphasize-actions.ts (hunk 1 above)
Comment (jakobe) [reply apps/cookbook/src/app/examples/header-example/examples/emphasize-actions.ts]: Nice catch! 👌🏻 
Fixed 👍🏻

File: libs/designsystem/action-group/src/action-group.component.ts (hunk 2 above)
Comment (jakobe) [reply libs/designsystem/action-group/src/action-group.component.ts:63]: Good point - added 👍🏻

**Review by RasmusKjeldgaard — APPROVED** [review]

**Pull request merged**